src/app/services/graph-client/auth/login-type.ts

```diff
@@ -0,0 +1,16 @@
+import { LoginType } from '../../../../types/enums';
```

Comment:
What do you think about adding util suffix to the class names utility classes. Example this can be renamed as `loginUtils.ts`